### PR TITLE
TP: 9035, Comment: Adds Karma to project

### PR DESCRIPTION
**Pull request by davidtrussler**
[TP: 9035](https://moneyadviceservice.tpondemand.com/restui/board.aspx?#page=task/9035&appConfig=eyJhY2lkIjoiOThBNEI1MURGODY0REREODQ4Qjk2MkE1RUY4NTRBMTIifQ==)

This PR adds Karma to the FinCap project to enable the addition of tests of JavaScript components.

### CODE REVIEW

**Comment by johnplayer1982** [comment]
Looks good to me :+1: however I dont think package-lock.json should be included?

File: package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "sinon": "^4.4.2"
+  },
+  "scripts": {
+    "test": "./test.sh"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/moneyadviceservice/fin_cap.git"
+  },
+  "author": "David Trussler",
```
Comment (margOnline) [inline package.json]: should the author be 'MAS Development Team'

**Comment by davidtrussler** [comment]
@johnplayer1982 @margOnline I wondered about package-lock.json - I think it should be included: that is the message returned when it's created and it is included on frontend.

File: package.json (hunk 1 above)
Comment (davidtrussler) [reply package.json]: I have no problem with that! (Added my name more or less automatically when prompted during the build process).

**Comment by tomas-stefano** [comment]
Following the documentation is good to add to source control: https://github.com/npm/npm/blob/v5.0.0/doc/files/package-lock.json.md

**Pull request merged**